### PR TITLE
Game Translation: Update guide with all recent features

### PR DESCRIPTION
**Pull request by Ghabry**

### CODE REVIEW

File: content/player/guide/game_translation.md [hunk 1]
```diff
@@ -20,18 +22,36 @@ The translation function in EasyRPG Player is similar to variant two, but is eas
 
 ### First Steps
 
-First create a new folder called ``languages`` in your game folder. In this folder you can now create further subfolders (one for each translation). The folder name does not matter. In the following we will call the folder ``LANGUAGE_NAME``.
+First create a new folder called ``Languages`` in your game folder. In this folder you can create further subfolders (one for each translation). The folder name does not matter. In the following we will call the folder ``LANGUAGE_NAME``.
```
Comment (Mimigris) [inline content/player/guide/game_translation.md:25]: If I'm not mistaken the "Languages" folder was renamed to "Language" in the latest continuous builds, so you may also have to change that in the guide.
screenshot_43

Comment (fdelapena) [reply content/player/guide/game_translation.md:25]: Also mentioned at lines 102, 104 and 152.

**Comment by fdelapena** [comment]
Jenkins: test this please :heart:

**Review by fdelapena — APPROVED** [review]
For other stuff besides the "Languages" issue, looks good to me.

**Comment by fdelapena** [comment]
The website check could pass after a rebase as well.

**Pull request merged**